libsolidity/codegen/ir/IRGeneratorForStatements.h

```diff
@@ -65,6 +65,8 @@ class IRGeneratorForStatements: public ASTConstVisitor
 
 	void setLValue(Expression const& _expression, std::unique_ptr<IRLValue> _lvalue);
 
+	Type const& type(Expression const& _expression) const;
+
```

Comment:
This doesn't need to be a member function, right?

Reply:
You mean it can be static? Right!